hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/server/ServerUtils.java

```diff
@@ -178,12 +184,90 @@ public static File getDirectoryFromConfig(ConfigurationSource conf,
             dbDirPath + " specified in configuration setting " +
             key);
       }
+      try {
```

Comment:
Can you update the javadoc of this method so that users of the method is aware it has side effects?

Reply:
Sure will do!!